src/app/portlets/dot-edit-page/main/dot-edit-page-main/dot-edit-page-main.component.html

```diff
@@ -1,2 +1,2 @@
 <router-outlet></router-outlet>
-<dot-edit-page-nav [pageState]="pageState | async"></dot-edit-page-nav>
\ No newline at end of file
+<dot-edit-page-nav [pageState]="pageState | async" (buttonClicked)="this.openPageProperties($event)"></dot-edit-page-nav>
```

Comment:
You shouldn't need a `this` here, if you need it maybe there is something to fix in the implementation.

Reply:
My questios is why `(buttonClicked)="this.openPageProperties($event)` instead of `(buttonClicked)="openPageProperties($event)`?

Reply:
typo